app/login/page.jsx

```diff
@@ -1,9 +1,30 @@
 import Link from 'next/link'
+import Logo from '@/components/Logo'
 
 export default function Login() {
   return (
     <main className="flex min-h-screen flex-col items-center">
-      <div>HelpRat Login</div>
+      <Logo />
+      <form>
+        <div className="relative mb-4">
+          <input type="text" id="username" placeholder="Username" />
+          <label htmlFor="username">Username</label>
+        </div>
+        <div className="relative mb-4">
+          <input type="password" id="password" placeholder="Password" />
+          <label htmlFor="password">Password</label>
+        </div>
+        <div className="mb-12 pb-1 pt-1 text-center">
+          <button>Log in</button>
+          <Link href="#!">Forgot password?</Link>
```

Comment:
forgot password should be a button element and grouped with the input fields and not the button